community/cypher/cypher-compiler-3.2/src/test/scala/org/neo4j/cypher/internal/compiler/v3_2/NotImplementedPlanContext.scala

```diff
@@ -23,11 +23,15 @@ import org.neo4j.cypher.internal.compiler.v3_2.spi._
 import org.neo4j.cypher.internal.frontend.v3_2.phases.InternalNotificationLogger
 
 class NotImplementedPlanContext extends PlanContext {
-  override def getIndexRule(labelName: String, propertyKey: String): Option[IndexDescriptor] = ???
+  override def indexesGetForLabel(labelId: Int): Iterator[IndexDescriptor] = ???
+
+  override def getIndexRule(labelName: String, propertyKeys: Seq[String]): Option[IndexDescriptor] = ???
 
   override def hasIndexRule(labelName: String): Boolean = ???
 
-  override def getUniqueIndexRule(labelName: String, propertyKey: String): Option[IndexDescriptor] = ???
+  override def uniqueIndexesGetForLabel(labelId: Int): Iterator[IndexDescriptor] = ???
+
+  override def getUniqueIndexRule(labelName: String, propertyKeys: Seq[String]): Option[IndexDescriptor] = ???
```

Comment:
Yeah, you are correct. I think we'll soon address index entry locking for MERGE. I think that we should, as part of that, take on this clean up as well. We should remove the concept of unique indexes from the Cypher code base.